src/ClientHandle.cpp

```diff
@@ -1532,6 +1539,27 @@ void cClientHandle::HandleSlotSelected(Int16 a_SlotNum)
 
 
 
+void cClientHandle::HandleSpectate(const AString & a_PlayerUUID)
```

Comment:
Do we have to add another method to cClientHandle to handle this? I'm not sure this is the best long term decision.